stubs/gdb/gdb/printing.pyi

```diff
@@ -0,0 +1,31 @@
+from collections.abc import Callable, Iterable
+
+import gdb
+from gdb import _PrettyPrinterLookupFunction
+
+class PrettyPrinter:
+
+    name: str
+    subprinters: list[SubPrettyPrinter]
+    enabled: bool
+
+    def __init__(self, name: str, subprinters: Iterable[SubPrettyPrinter] | None = ...) -> None: ...
+    def __call__(self, val: gdb.Value) -> gdb._PrettyPrinter | None: ...
+
+class SubPrettyPrinter:
+
+    name: str
+    enabled: bool
+
+    def __init__(self, name: str) -> None: ...
+
+GenPrinterFunction = Callable[[gdb.Value], PrettyPrinter]
```

Comment:
Same here -- did you mean for this to be a type alias, or did you mean to do `GenPrinterFunction: Callable[[gdb.Value], PrettyPrinter]`

Reply:
This name doesn't appear in the gdb source code, I think it should be a private type alias.

Reply:
I definitely meant it as a private type alias, but forgot about it when writing the related part in `__init__.py:187`. I’ll just remove it, thanks! Fix also in #8144